src/btwa_api/app/detection.py

```diff
@@ -0,0 +1,84 @@
+SCAN_PERIOD = 2 * 60 * 1000  # 2 minutes in milliseconds
+BREAK_MEETING_TRESHOLD = 60 * 60 * 1000  # 1 hour in milliseconds
+SCORE_DEDUCTION_ON_MEETING_BREAK = 3
+INFECTED_SCORE_TRESHOLD = 100
+
+
+def detect(input_data):
```

Comment:
I would vote for this function to accept just a single row, or better, just relevant arguments. At this moment, it's coupled with the whole CSV structure which is not very future-proof. My vision is to call `detect` method from elsewhere, passing it just relevant data, no matter where they come from.

Reply:
OK, let it be dictionary as input.

Reply:
The thing is, detection is based on multiple rows - we need more "meeting" occurrences to detect potential infection.

We don't really know, what might be relevant data, so I'd accept the full row and eventually update just the algorithm rather than input + the algorithm.

Discussed already on Slack, I'm resolving the comment.